homeassistant/components/nina/strings.json

```diff
@@ -11,7 +11,8 @@
           "_r_to_u": "City/county (R-U)",
           "_v_to_z": "City/county (V-Z)",
           "slots": "Maximum warnings per city/county",
-          "headline_filter": "Blacklist regex to filter warning headlines"
+          "headline_filter": "Blacklist regex to filter warning headlines",
+          "area_filter": "Whitelist regex to filter warnings based on affected areas"
```

Comment:
These are very long for labels.
Split to data (a short label, maybe just "Blacklist regex" + "Whitelist regex") + data_description (The existing data should do).